docs/root/start/sandboxes/locality_load_balancing.rst

```diff
@@ -0,0 +1,141 @@
+    $ docker-compose exec -T client-envoy curl -s locality-load-balancing_backend-local-2_1:8000/healthy
+
+
+Step 3: Scenario with multiple replica in the highest priority locality
+***********************************************************************
+
+Continue from previous step. We first scale up the ``local-1`` locality to 5 replicas.
+
+.. code:: shell
+
+    $ docker-compose up --scale backend-local-1=5 -d
+
+We are going to show the scenario that ``local-1`` is just partially healthy. So let's bring down 4 of the replicas in ``local-1``.
+
+.. code:: shell
+
+    # bring down local-1 replicas
+    $ docker-compose exec -T client-envoy curl -s locality-load-balancing_backend-local-1_2:8000/unhealthy
+    $ docker-compose exec -T client-envoy curl -s locality-load-balancing_backend-local-1_3:8000/unhealthy
+    $ docker-compose exec -T client-envoy curl -s locality-load-balancing_backend-local-1_4:8000/unhealthy
+    $ docker-compose exec -T client-envoy curl -s locality-load-balancing_backend-local-1_5:8000/unhealthy
+
+    # check healthiness
+    $ docker-compose exec -T client-envoy curl -s localhost:8001/clusters | grep health_flags
+
+    backend::172.28.0.4:8000::health_flags::/failed_active_hc
+    backend::172.28.0.2:8000::health_flags::/failed_active_hc
+    backend::172.28.0.5:8000::health_flags::/failed_active_hc
+    backend::172.28.0.6:8000::health_flags::/failed_active_hc
+    backend::172.28.0.7:8000::health_flags::healthy
+    backend::172.28.0.8:8000::health_flags::healthy
+    backend::172.28.0.3:8000::health_flags::healthy
+
+You should see that 4 backend endpoints become unhealthy.
+
+Now we send the 100 requests again.
+
+.. code:: shell
+
+    # watch traffic change
+    $ docker-compose exec -T client-envoy python3 client.py http://localhost:3000/ 100
+
+    Hello from backend-remote-1!: actual weight 37.0%
+    Hello from backend-local-2!: actual weight 36.0%
+    Hello from backend-local-1!: actual weight 27.0%
+    Failed: 0
+
+As ``local-1`` does not have enough healthy workloads, requests are partially shared by secondary localities.
+
+If we bring down all the servers in priority 1 locality, it will make priority 1 locality 0% healthy. The traffic should split between priority 0 and priority 2 localities.
+
+.. code:: shell
+
+    $ docker-compose exec -T client-envoy curl -s locality-load-balancing_backend-local-2_1:8000/unhealthy
+    $ docker-compose exec -T client-envoy curl -s locality-load-balancing_backend-remote-1_1:8000/unhealthy
+    $ docker-compose exec -T client-envoy python3 client.py http://localhost:3000/ 100
+
+    Hello from backend-remote-2!: actual weight 77.0%
+    Hello from backend-local-1!: actual weight 23.0%
+    Failed: 0
+
+**Conclusion:** When the healthiness of a locality drops below a threshold, the next priority locality will start to share the traffic. The default overprovisioning factor is 1.4, which means that the shifting healthiness threshold is at around 71%.
```

Comment:
im wondering about the wording of this - maybe something like

> When the healthiness of the backends with a given priority drops below a threshold, the next priority...

(also please avoid using bold for a heading - in this case im wondering if this bit would be better in the introduction)

Reply:
Great suggestion. Will move it to the intro section